### PR TITLE
Make sure we log stuff using logger

### PR DESCRIPTION
**Pull request by suyograo**
Partially addresses #5975

### CODE REVIEW

File: logstash-core/lib/logstash/runner.rb [hunk 1]
```diff
@@ -191,21 +194,27 @@ def execute
     # override log level that may have been introduced from a custom log4j config file
     LogStash::Logging::Logger::configure_logging(setting("log.level"))
 
+    # Adding this here because a ton of LS users install LS via packages and try to manually 
+    # start Logstash using bin/logstash. See #5986. I think not logging to console is good for 
+    # services, but until LS users re-learn that logs end up in path.logs, we should keep this 
+    # message. Otherwise we'll be answering the same question again and again.
+    puts "Sending Logstash logs to #{setting("path.logs")} which is now configured via log4j2.properties."
+
```
Comment (jordansissel) [inline logstash-core/lib/logstash/runner.rb]: ++++

**Comment by jordansissel** [comment]
I like these changes. LGTM.

**Comment by jsvd** [comment]
Looking at the failure I see no relation to this PR's code. I created issue https://github.com/elastic/logstash/issues/6003 to address the test's flakyness

**Comment by suyograo** [comment]
Thanks @jsvd. The re-run has passed, so I am going to merge this.

**Pull request closed without merging**